File: 389-FindTheDifference/389-FindTheDifference.java
// Last updated: 2/22/2026, 1:54:55 PM
class Solution {
    public char findTheDifference(String s, String t) {
        char result = '\0';
        for(char ch: s.toCharArray()) result^=ch;
        for(char ch:t.toCharArray()) result^=ch;
        return result;
    }
}

File: 389-FindTheDifference/389-FindTheDifference.cpp
// Last updated: 3/13/2026, 8:17:12 PM
class Solution {
public:
    char findTheDifference(string s, string t) {
        char result = '\0';
        for(char ch : s) result^=ch;
        for(char ch : t) result^=ch;
        return result;
    }
};